CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 ## June 2018
 
+* Arguments for `skt merge` can now be used multiple times instead of
```

Comment:
I would change this to "should now be used multiple times", and to "supplying multiple values".